src/wp-admin/user-edit.php

```diff
@@ -621,9 +621,9 @@
 									<?php
 									if ( IS_PROFILE_PAGE ) {
 										$description = sprintf(
-											/* translators: %s: Gravatar URL. */
+											/* translators: %s: The localized Gravatar URL. */
 											__( '<a href="%s">You can change your profile picture on Gravatar</a>.' ),
-											__( 'https://en.gravatar.com/' )
+											__( 'https://gravatar.com/' )
```

Comment:
The localization note can go in the URL. 

```suggestion
											/* translators: The localized Gravatar URL. */
											__( 'https://gravatar.com/' )
```

Reply:
@peterwilsoncc Thanks for the review I have implemented the suggestions.